src/main/java/com/redhat/devtools/lsp4ij/console/actions/AutoFoldingAction.java

```diff
@@ -69,4 +70,9 @@ public void setSelected(@NotNull AnActionEvent e, boolean state) {
             }
         });
     }
+
+    @Override
+    public @NotNull ActionUpdateThread getActionUpdateThread() {
+        return ActionUpdateThread.EDT;

```

Comment:
I did that because https://github.com/JetBrains/intellij-community/blob/164e6d70fdf689d4e30735e1c2b8418c31238183/platform/execution-impl/src/com/intellij/execution/actions/ClearConsoleAction.java#L22 does that.

But I don't masterize that.